vtparse/src/lib.rs

```diff
@@ -894,8 +894,10 @@ mod test {
 
     #[test]
     fn test_csi_too_many_params() {
+        // To be honest, I'm not sure this test is meaningful anymore
+        // now that MAX_PARAMS is so high...
         assert_eq!(
-            parse_as_vec(b"\x1b[0;1;2;3;4;5;6;7;8;9;0;1;2;3;4;51;6p"),
+            parse_as_vec(b"\x1b[0;1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16;17;18;19;20;21;22;23;24;25;26;27;28;29;30;31;32;33;34;35;36;37;38;39;40;41;42;43;44;45;46;47;48;49;50;51;52;53;54;55;56;57;58;59;60;61;62;63;64;65;66;67;68;69;70;71;72;73;74;75;76;77;78;79;80;81;82;83;84;85;86;87;88;89;90;91;92;93;94;95;96;97;98;99;100;101;102;103;104;105;106;107;108;109;110;111;112;113;114;115;116;117;118;119;120;121;122;123;124;125;126;127;128p"),
```

Comment:
what I'd suggest here is computing both the input string and the expected result in a loop, something like:

```rust
let mut input = "\x1b[0".to_string();
let mut params = vec![CsiParam::Integer(0)];

for n in 1..=128 {
  input.push_str(&format!(";{n}"));
  params.push(CsiParam::P(b';'));
  params.push(CsiParam::Integer(n));
}

input.push('p');
params.push(CsiParam::P(b';'));
```

Reply:
Yeah, that's way nicer than my naive copy-paste. I'll do that.

I'm a bit of a noob at Rust, in case you couldn't tell. 😊

Reply:
Done. This didn't quite work as is, but it wasn't far off :)